packages/dx-react-grid-bootstrap4/src/templates/table-group-cell/cell.jsx

```diff
@@ -1,12 +1,15 @@
 import * as React from 'react';
 import * as PropTypes from 'prop-types';
 import classNames from 'classnames';
+import { argumentsShallowEqual } from '@devexpress/dx-core';
 
-export const Cell = ({
+export const Cell = React.memo(({
```

Comment:
`children` property will be different (most likely) on each new render, won't it?

Reply:
That's right, we shouldn't use memo here